infra/terraform/test-org/ci-project/gar.tf

```
@@ -0,0 +1,19 @@
+module "artifact_registry" {
+  source  = "GoogleCloudPlatform/artifact-registry/google"
+  version = "~> 0.1"
```

nit: any reason not to start with?
```suggestion
  version = "~> 0.2"
```